my-app/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+/* To learn more about this file see: https://angular.io/config/tsconfig. */
+{
+  "compileOnSave": false,
```

Comment:
I would recommend to enable strict mode from the beginning, see https://blog.angular.io/angular-cli-strict-mode-c94ba5965f63